pkg/apis/eventing/v1/trigger_types.go

```diff
@@ -83,8 +83,22 @@ type TriggerSpec struct {
 	// +optional
 	Filter *TriggerFilter `json:"filter,omitempty"`
 
-	// Subscriber is the addressable that receives events from the Broker that pass the Filter. It
-	// is required.
+	// Filters is an experimental field that conforms to CNCF CloudEvents Subscriptions
+	// API. It's An array of filter expressions that evaluates to true or false.
+	// If any filter expression in the array evaluates to false, the event MUST
+	// NOT be sent to the Subscriber. If all the filter expressions in the array
+	// evaluates to true, the event MUST be attempted to be delivered. Absence of
+	// a filter or empty array implies a value of true. In the event of users
+	// specifying both Filter and Filters, then the later will override the first.
+	// This will allow our users to try out the effect of the new filters field
+	// without compromising existing Filter objects and try it out on existing
+	// Trigger objects.
```

Comment:
Same comment about "object" meaning two different things, I think this deserves a minor rephrasing.